hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/HoodieMultiTableDeltaStreamer.java

```diff
@@ -378,16 +383,23 @@ private static String resetTarget(Config configuration, String database, String
   /**
    * Creates actual HoodieDeltaStreamer objects for every table/topic and does incremental sync.
    */
-  public void sync() {
-    for (TableExecutionContext context : tableExecutionContexts) {
```

Comment:
@watermelon12138 the execution is happening serially in the current implementation. Please see this for loop here. We intend to change it to parallel execution.